scripts/upower/95-upower-hid.hwdb

```diff
@@ -0,0 +1,200 @@
+##############################################################################################################
+# Uninterruptible Power Supplies with USB HID interfaces
+#
+# This file was automatically generated by NUT:
+# https://github.com/networkupstools/nut/
+#
+# To keep up to date, monitor upstream NUT
+# https://github.com/networkupstools/nut/commits/master/scripts/upower/95-upower-hid.hwdb
+# or checkout the NUT repository and call 'tools/nut-usbinfo.pl'
+
+# Hewlett Packard
+usb:v03f0p0001*
+usb:v03f0p1f06*
+usb:v03f0p1f08*
+usb:v03f0p1f09*
+usb:v03f0p1f0a*
+usb:v03f0p1fe0*
+usb:v03f0p1fe1*
+usb:v03f0p1fe2*
+usb:v03f0p1fe3*
+usb:v03f0p1fe5*
+usb:v03f0p1fe6*
+usb:v03f0p1fe7*
+usb:v03f0p1fe8*
+ UPOWER_BATTERY_TYPE=ups
+ UPOWER_VENDOR=Hewlett Packard
```

Comment:
I am not sure about the hwdb format; does it apply the `UPOWER_*` details to just one `usb:*` line above it, or to all lines above? If just one, then I think it does not suffice - the other IDs are also UPSes of different models (and possibly protocol revisions) from the same vendor.